pure2.md

```diff
@@ -0,0 +1,535 @@
+
+Offline-update targets metadata files SHOULD NOT be listed in the main snapshots metadata file of the Director repository, if implemented. Because the offline-update role signs its own snapshot metadata, listing those files in the main snapshots metadata file is redundant, and increases the size of a file that is downloaded very frequently.
+
+### Required files for offline updates
+
+The following elements SHALL be present on any offline-update storage location (in a "well-known" location configured in advance on the on the Primary):
+
+-   Image repository metadata:
+
+    -   All Root metadata files from the Image repository, from version 1 to the most-recent version at the time the offline update is generated
+
+    -   The latest version of every Targets metadata file, including delegations, at the time the offline update is generated
+
+    -   The latest version of the Snapshot metadata file at the time the offline update was generated
+
+-   Director repository metadata:
+
+    -   All Root metadata files from the Director repository, from version 1 to the most-recent version at the time the offline update is generated
+
+    -   The offline-update Snapshot metadata file
+
+    -   Exactly one offline-update targets metadata file
+
+-   All target images referenced in the offline-update targets metadata file
+
+### Metadata verification procedures
+
+The procedures for verifying offline-update metadata are similar to the procedures for verifying normal director metadata. However, there are some key differences. The procedures for sending images from the primary to secondaries, as well as for verifying and installing the images once the metadata has been verified, remain the same.
+
+Note: wherever the "Uptane Standard" or "the Standard" is referenced, it refers to version 1.2.0 of the Standard, available at https://uptane.github.io/uptane-standard/1.2.0/uptane-standard.html.
+
+The primary still SHALL verify all target images listed in the Director offline-update metadata, while secondaries MAY verify only the images they will install.
+
+#### Download and verify metadata from the well-known location
+
+The Primary SHALL download metadata for all offline-update targets and perform a full verification on it. Full verification of metadata means that the ECU checks that the Targets metadata about images from the Director repository matches the Targets metadata about the same images from the Image repository. This provides resilience to a key compromise in the system.
+
+Because of the nature of offline updates, sometimes the offline-update image will contain newer metadata than the primary has, and sometimes the converse will be true. Most of the differences or new steps in the verification process are to deal with this reality; the rest of the differences are steps that were removed because they aren't applicable, mostly relating to the ECU version reports and vehicle version manifest.
```

Comment:
```suggestion
Because of the nature of offline updates, sometimes the offline-update image will contain newer metadata than the primary has, and sometimes the converse will be true. Most of the differences or new steps in the verification process are to deal with this reality; the rest of the differences are steps that were removed because they aren't applicable, in particular relating to the ECU version reports and vehicle version manifest.
```
Something about "mostly" sounds too hand-wavy to me.